tests/opae-c/test_pluginmgr_c.cpp

```diff
@@ -486,6 +486,7 @@ TEST_P(pluginmgr_c_p, fpgaReset_null_handle) {
   opae_plugin_mgr_finalize_all();
 }
 
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(pluginxtest_c_p);
```

Comment:
Shouldn't this be pluginmgr_c, instead of pluginxtest_c_p?

Reply:
fixed